### DISCUSSION TITLE
How does onLeft cancel the default event, that is, without changing the cursor position of the input field

### DISCUSSION THREAD

**Discussion by alkene5** [johnlindquist/kit#1319] [Q&A] [answered]
```js
await arg({
  onLeft() {},  // The input field cursor position is changed by default, but I don't want it to change
  onRight() {},
}, ...)

**Comment by johnlindquist** [comment, marked as answer]
@alkene0005 

That's not currently supported.

Unfortunately, it's very challenging to have the script tell that app to "prevent" events. Because the script doesn't know about the events until after they've happened. I'll keep trying to think of some sort of configuration you could pass in, but I don't have any plans at this time.

**Reply by alkene5** [reply]
ok
